Source/Core/Core/IOS/IPC.h

```diff
@@ -59,6 +59,12 @@ void Shutdown();
 bool Reload(u64 ios_title_id);
 u32 GetVersion();
 
+// Set or get the PPC's UID/GID.
```

Comment:
This is kind of unnecessary, given the functions themselves already say the equivalent of this.